pkgs/top-level/python-packages.nix

```diff
@@ -15595,6 +15595,10 @@ in {
   pika-pool = callPackage ../development/python-modules/pika-pool { };
   platformio = callPackage ../development/python-modules/platformio { };
 
+  pykms = callPackage ../development/libraries/kmsxx {
+    pythonBindings = python;
```

Comment:
The manual says

> # Dependencies:
> - libdrm
> - Python 3.x (for python bindings)

...so it should be disabled for Python 2.x.

---

Nevermind: https://github.com/tomba/kmsxx/commit/643d678146f0e292ed211e9fe214c145e697d2d4